examples/plot_mandrill.py

```diff
@@ -0,0 +1,89 @@
+"""
+DiCoDiLe on the Mandrill image
+==============================
+
+This example illlustrates reconstruction of `Mandrill image
+<http://sipi.usc.edu/database/download.php?vol=misc&img=4.2.03>`_
+using DiCoDiLe algorithm with soft_lock "corner" and 16 workers.
+
+"""  # noqa
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+from dicodile.data.images import fetch_mandrill
+
+from dicodile.utils.dictionary import init_dictionary
+from dicodile.utils.viz import display_dictionaries
+from dicodile.utils.csc import reconstruct
+
+from dicodile import dicodile
+
+
+###############################################################################
+# We will first download the Mandril image.
+
+X = fetch_mandrill()
+
+plt.axis('off')
+plt.imshow(X.swapaxes(0, 2))
+
+
+###############################################################################
+# We will create a random dictionary of K = 25 patches of size 8 x 8 from the
+# original Mandrill image to be used for sparse coding.
+
+# set dictionary size
+n_atoms = 25
+
+# set individual atom (patch) size
+atom_support = (4, 4)
+
+# random state to seed the random number generator
+rng = np.random.RandomState(60)
+
+D_init = init_dictionary(X, n_atoms, atom_support, random_state=rng)
+
+###############################################################################
+# Set parameters.
+
+w_world = 3
+n_workers = w_world * w_world
+
+###############################################################################
+# Run DiCoDiLe
+
+pobj, times, D_hat, z_hat = dicodile(X, D_init, n_iter=3,
+                                     n_workers=n_workers,
+                                     strategy='greedy',
+                                     dicod_kwargs={"max_iter": 10000},
+                                     verbose=20)
+
+
+z_hat = np.clip(z_hat, -1e3, 1e3)
+print("[DICOD] final cost : {}".format(pobj))
+
+###############################################################################
+# Plot the dictionary patches
+
+list_D = np.array([D_hat])
+display_dictionaries(*list_D)
```

Comment:
This should be sufficient no?
```suggestion
display_dictionaries(D_hat)
```

Or maybe `display_dictionaries(D_init, D_hat)` to show how it moved?

Reply:
This is how it seems:

![dicts](https://user-images.githubusercontent.com/2099645/107764875-32521f80-6d31-11eb-873d-2281c21de548.png)

Which one would you prefer?

Reply:
can you renormalize by the max `D_hat / D_hat.max()`, It think this should improve the visualisation.

Reply:
This seems much nicer. Shall I keep both?

![dicts_2](https://user-images.githubusercontent.com/2099645/107775818-1b1b2e00-6d41-11eb-8881-a554acf2a296.png)

Reply:
I think just this one is fine. Thx a lot this look very good now.